### PR TITLE
Add fonts

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: globie-library/type.styl [hunk 1]
```diff
@@ -1,12 +1,3 @@
-// FONTS
-
-// declare font families here
-
-$font-sans = font-stack('Helvetica');
```
Comment (varffs) [inline globie-library/type.styl:5]: yh we need to move this out of the lib and into the main.styl file

File: src/styl/fonts/fonts.styl [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-BookItalic.eot');
+	src: url('../static/fonts/PostGrotesk-BookItalic.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-BookItalic.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-BookItalic.ttf') format('truetype');
+	font-weight: normal;
+	font-style: italic;
```
Comment (varffs) [inline src/styl/fonts/fonts.styl:44]: Book Italic not Regular Italic?

File: src/styl/fonts/fonts.styl [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+// YoungSerif
+
+@font-face {
+	font-family: 'YoungSerif';
+	src: url('../static/fonts/YoungSerif-Regular.eot');
+	src: url('../static/fonts/YoungSerif-Regular.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/YoungSerif-Regular.woff') format('woff'),
+		url('../static/fonts/YoungSerif-Regular.ttf') format('truetype');
+	font-weight: 600;
```
Comment (varffs) [inline src/styl/fonts/fonts.styl]: Why not keep this normal? If there is only one weight will it not always need the `font-weight` value set when this typeface is used.

File: src/styl/site.styl [hunk 4]
```diff
@@ -60,6 +71,15 @@ body
 .line-larger
   line-height: $font-leading-larger
 
+.font-logo
+  font-family: $font-logo
+
+.font-ruberta
```
Comment (varffs) [inline src/styl/site.styl:77]: is this font only ever used for the text that says Ruberta? Are they any other contexts for it? If not perhaps we could do a extreme subset down to just those letters, or at least we should do some subsetting.

File: src/styl/fonts/fonts.styl [hunk 5]
```diff
@@ -0,0 +1,89 @@
+/* This stylesheet generated by Transfonter (https://transfonter.org) on August 30, 2017 7:40 PM */
+
+// Druk
+
+@font-face {
+	font-family: 'Druk';
+	src: url('../static/fonts/Druk-Bold.eot');
+	src: url('../static/fonts/Druk-Bold.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/Druk-Bold.woff') format('woff'),
```
Comment (varffs) [inline src/styl/fonts/fonts.styl:9]: I think we should always check the box for woff2 on Transfonter (I don't understand why it isn't default) as woff2 files are about 1/3 of the size of woff files and widely supported

File: src/styl/fonts/fonts.styl (hunk 2 above)
Comment (mikeyrayvon) [reply src/styl/fonts/fonts.styl:44]: yea its actually regular italic. the file name was wrong

File: src/styl/site.styl (hunk 4 above)
Comment (mikeyrayvon) [reply src/styl/site.styl:77]: idk what you mean by subsetting but i'll make a task

**Pull request merged**